pkg/client/typed/dynamic/client_pool.go

```diff
@@ -28,16 +29,21 @@ import (
 
 // ClientPool manages a pool of dynamic clients.
 type ClientPool interface {
```

Comment:
Yes, will do.

On Tue, Sep 13, 2016 at 9:26 PM, Chao Xu <notifications@github.com> wrote:

> In pkg/client/typed/dynamic/client_pool.go
> <https://github.com/kubernetes/kubernetes/pull/32549#discussion_r78673815>
> :
>
> > @@ -28,16 +29,21 @@ import (
> >
> >  // ClientPool manages a pool of dynamic clients.
> >  type ClientPool interface {
>
> Can you add a comment to clientPoolImpl saying it reuses restclient for
> resources/kind of the same group version? It's less obvious after the
> interface changes.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/32549/files/3f30b865dfde4d204272ad2d7cd9c6edc41a5448#r78673815>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABG_p7XNJ5SwMhUMn4VFX73u00eESxklks5qp01OgaJpZM4J7Qa7>
> .
>